_data/en/settings.yml

```diff
@@ -234,545 +148,12 @@ help_page:
-          grant applications to fill out later, then return repeatedly to check the
-          application status? Or will they be a one-time or infrequent, as when people
-          download medical or financial records?
-      li_2:
-        content: What kind of information do you need to protect your customers?
-        p_1: Do you need full name and other personal information so that users can
-          access private information? Or do you only need to verify that a user fits
-          in certain categories, such as the veterans category or the senior citizens
-          category?
-      li_3:
-        content: What sort of crime might access to this information make possible?
-        p_1: Information that seems innocent on its own might still be valuable to
-          fraudsters and other criminals in combination with other easily accessed
-          information.
-      li_4:
-        content: What other means of security are available?
-        p_1: Postal tracking numbers, for example, are not secrets because the package
-          will only be delivered to a specific address. The safety of the delivery
-          rests on the security of the building and the conduct of the delivery person,
-          not on the secrecy of the number itself.
-    ul_3:
-      li_1:
-        content: What resources are unique to your agency?
-        p_1: Individuals’ confidential interactions with government agencies can generate
-          a trail of metadata. Used carefully, that metadata can facilitate identity
-          verification based on knowledge of those activities. Other government organizations
-          serve as authoritative repositories of biometric data available for internal
-          use. Some agencies may have physical locations that customers can visit.
-    ul_4:
-      li_1:
-        span_1: Pre-filling online forms with verified information speeds up application
-          processing.
-        span_2: There’s less redundant effort, and users don’t need to worry about
-          basic errors.
-      li_2:
-        span_1: Authenticated users can access and download data the system holds
-          about them, such as account activity.
-        span_2: With a verified legal identity, the user can access very sensitive
-          medical or financial records and even download them.
-      li_3:
-        span_1: Identity systems can protect your privacy.
-        span_2: If you need to be 21 or older to access a service, you can authorize
-          an identity system to confirm your age without sharing your exact birth
-          date.
-    ul_5:
-      li_1:
-        a_1: Please read the developer documentation.
-    ul_6:
-      li_1:
-        a_1: National Institute of Standards and Technology
-        content: '(NIST 800-63-3)'
-      li_2:
-        a_1: Digital Services Playbook
-      li_3:
-        a_1: GitHub repo for login.gov
-      li_4:
-        a_1: On the Internet, nobody knows you're a dog
 index:
```

Comment:
Could we safely remove `index.column_1`, `index.column_2`, `index.column_3`, `index.intro`?

Reply:
Yes! @jmdembe can you remove the `index:` content from all three yaml.